engine/ekf.py

```diff
@@ -103,20 +103,20 @@ def get_h_jac(self, pose):
         """
         return np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
 
-    def get_arc_lengths(self, left_side_w, right_side_w):
+    def get_arc_lengths(self, left_motor_revolutions, right_motor_revolutions):
         """
         Returns the arc_lengths of the left side of the robot and the right side of the robot 
         given the angular velocity of the left side and angular velocity of the right side. 
 
         left_side_w: angular velocity of the motors on the left side of the robot
         right_side_w: angular velocity of the motors on the right side of the robot
         """
-        # TODO: from electrical: from motors, get angular velocity
-        # TODO: from angular velocity, find arc_lengths 
+        # TODO: from electrical: from motors, number of revolutions
+        # TODO: from number of revolutions, using the circumference of the wheels, figure out the distance traveled 
         # Note: this was put on hold Spring '23 because Professor advised that getting velocity
         # in this fashion was not that accurate 
-        left_arc_length = left_side_w # TEMP
-        right_arc_length = right_side_w # TEMP
+        left_arc_length = left_motor_revolutions # this will be some sort of relation with the wheel circumference
```

Comment:
I believe its (revolution since last call) * 2 * pi * wheel radius

Reply:
will keep this in mind when communicating w electrical for this function!